### DISCUSSION TITLE
A use for nether wart?

### DISCUSSION THREAD

**Discussion by Aceplante** [vectorwing/FarmersDelight#698] [Ideas]
There could be a recipe using nether wart and there currently does not appear to be. Maybe a "Nether stew"?

**Comment by vectorwing** [comment]
Not every "organic" item needs to be part of a meal. Nether Wart seems (to me, at least) like the kind of ingredient you wouldn't put on your food, but which you could extract some essence out of (such as Potions).

Potion ingredients in MC, in general, are intended to have that classic "creepy cauldron" aesthetic, hence why they seem so off-beat compared to normal food.
